jazkarta/shop/browser/checkout.py

```diff
@@ -24,13 +32,318 @@
 from ..vocabs import country_names
 
 
-@implementer(IStripeEnabledView)
 class CheckoutForm(BrowserView):
+
+    def __call__(self):
+        payment_processor = get_setting('payment_processors')
+        if payment_processor != []:
+            p_p = payment_processor[0]
```

Comment:
This is a list of enabled payment processors, we can only use one at a time, so I just pick the 1st one if someone accidentally happens to add more than one to the enabled list.